.github/workflows/ci.yml

```diff
@@ -78,7 +78,7 @@ jobs:
       run: |
         sudo apt-get update
         python -m pip install --upgrade pip
-        python -m pip install copier
+        python -m pip install copier mypy
```

Comment:
I'm not sure this is necessary. I think that the dependency should be installed when we call pip install .[dev] a couple steps later.

Reply:
I put it in because my trial pull (done in a personal github) failed to find it!
that said, I do not understand WHY it wasn't found. Maybe it was a transient failure?
I've had trouble with github in the last 48 hours with several uses of the template timing out and so it is a bit difficult to untangle.